app/assets/javascripts/menu.js

```diff
@@ -32,7 +32,6 @@ function initMenu() {
     }
   };
 }
-
-window.onload = () => {
+window.addEventListener("onload", () => {
```

Comment:
Let's use the `"DOMContentLoaded"` event here and below instead, since we just need the DOM to be loaded. (https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event)